utils.lua

```diff
@@ -50,5 +50,9 @@ folly_utils = {
         Fatal = function(message)
             sendFatalMessage(message, "Folly Jokers | Fatal");
         end,
-    }
+    },
+    
+    lerp = function(a, b, t)
+        return a + (b - a) * t
+    end
```

Comment:
```suggestion
    pseudorandom_float = function(seed, min, max)
        return min + pseudorandom(seed)  * (max - min);
    end
```
any reason to have a general lerp instead doing pseudorandom for floats?
should function the same but takes seed instead of t.
parameter ordering is same as pseudorandom